src/middleware/class/class.ts

```diff
@@ -0,0 +1,5 @@
+export class JWTPayload {
```

Comment:
Can remove this, the exact same interface is available as `UserDecoded` in `auth.ts`